contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -0,0 +1,298 @@
+                    </doc>
+                    <enumeration>
+                        <item value="x-ray"/>
+                        <item value="ultraviolet"/>
+                        <item value="visible light"/>
+                    </enumeration>
+                </field>
+            </group>
+            <group type="NXbeam">
+                <field name="distance" type="NX_NUMBER">
+                    <doc>
+                         Distance of the point of evaluation of the beam from the sample surface.
+                    </doc>
+                </field>
+                <field name="incident_energy" type="NX_FLOAT"/>
+                <field name="incident_energy_spread" recommended="true" type="NX_NUMBER"/>
+                <field name="incident_polarization" recommended="true" type="NX_NUMBER"/>
+            </group>
+            <group type="NXelectronanalyser">
+                <field name="description" type="NX_CHAR"/>
+                <field name="energy_resolution" recommended="true" type="NX_FLOAT">
+                    <doc>
+                         Energy resolution of the analyser with the current setting. May be linked from a
+                         NXcalibration.
+                    </doc>
+                </field>
+                <field name="fast_axes" recommended="true" type="NX_CHAR"/>
+                <field name="slow_axes" recommended="true" type="NX_CHAR"/>
+                <group type="NXcollectioncolumn">
+                    <field name="scheme" type="NX_CHAR">
+                        <doc>
+                             Scheme of the electron collection column.
+                        </doc>
+                        <enumeration>
+                            <item value="Standard"/>
+                            <item value="Angular dispersive"/>
+                            <item value="Selective area"/>
+                            <item value="Deflector"/>
+                            <item value="PEEM"/>
+                            <item value="Momentum Microscope"/>
+                        </enumeration>
+                    </field>
+                    <field name="mode" recommended="true" type="NX_CHAR"/>
+                    <field name="projection" recommended="true" type="NX_CHAR"/>
+                    <group name="field_aperture" optional="true" type="NXaperture">
+                        <doc>
+                             The size and position of the field aperture inserted in the column. To add
+                             additional or other apertures use the APERTURE group of NXcollectioncolumn.
+                        </doc>
+                    </group>
+                    <group name="contrast_aperture" optional="true" type="NXaperture">
+                        <doc>
+                             The size and position of the contrast aperture inserted in the column. To add
+                             additional or other apertures use the APERTURE group of Nxcollectioncolumn.
```

Comment:
capitalize `NX` here in `NXcollectioncolumn`

Reply:
Typo is corrected